tsc/meetings/2020-08-11.md

```diff
@@ -0,0 +1,69 @@
+Minutes from 60th OpenVDB TSC meeting, Aug 11th, 2020, (EDT)
+
+Attendees: *Nick* A., *Jeff* L., *Ken* M., *Dan* B.
+
+Additional Attendees: JT Nelson (Blender), Andre Pradhana (DW)
+
+Regrets: *Peter* C.
+
+Agenda:
+
+1) Confirm quorum
+2) Secretary
+3) Forum
+4) Fast Sweeping
+5) 7.1 Release
+6) NanoVDB
+7) Next Meeting
+
+
+1) Quorum was confirmed.
+
+2) Secretary was Dan Bailey
+
+3) Forum
+
+Request to remove v3.3.0 branch, Dan has done this. Request to improve
+benchmarks, this would be good to do. Request to move INVALID_IDX definition to
+header, it is believed that this is indicative of a deeper linking issue rather
+than an issue in itself, Nick to reply to question.
+
+4) Fast Sweeping
+
+Nick advocating removing testing logic from the PR. Ken to leave in until this
+feature has matured slightly, then tidy up. Brief discussion about unit testing
+in general as it is solving many purposes. Would be worth separating out
+benchmarks, regression tests, sample code and debugging data from unit tests.
+
+In case of no zero crossing in all or part of a grid, Jeff wants the same
+topology but different values. Never useful to know if sweep did something or
+not. Algorithm should return original grid not an empty grid. Ken agreed and
+will revisit this.
+
+Andre asked about what should happen at the SOP level if a vector field is
+passed in to be used for zero crossing. Should produce an error in this case.
+
+Fast sweeping SOP can either handle pairs of grids containing one SDF and one
+extension field or one SDF with multiple extension fields. Not clear what to do
+here. Best option seems to be to produce a warning if more than one SDF is
+passed in.
+
+5) 7.1 Release
+
+Dan to do 7.1 release as Ken and Nick are preparing for Open Source Day. Last
+two PRs are 746 (fast sweeping) and 775 (openexr 2.5 fix). Nick to merge 775,
+Ken / Dan to look at DCO issue and then once merged, 7.1 can be released.
+
+Decision to hold off on merging other PRs until after 7.1. Next time, a separate
+release branch would be preferable here.
+
+6) NanoVDB
+
+Question about how to deploy. Jeff and Dan favour doing a lightweight
+integration into OpenVDB. All agreed that this is the best choice in light of
```

Comment:
can you elaborate on what "lightweight integration means". My recollection is that we where debating two different approaches 1) adding NanoVDB into the existing opevenvdb repository as a "feature branch" or 2) Initially releasing NanoVDB as a separate repository and then integrate it into openvdb later. Jeff and Dan were in favor of 1), whereas Nick and Ken preferred 2). We agreed to continue this discussion at the next meeting.

Reply:
I can clarify that - what I meant by lightweight integration was the "feature branch" option. 

Initially, yes I believe we were divided on how to proceed here. However, it was my understanding that the deciding factor here was the realization that NanoVDB would have to be hosted by the ASWF. For the separate repo option (2), we would thus need to request a new NanoVDB ASWF repo which would also require setting up CLAs, license agreements, etc. Finally, we coalesced around the feature branch option (1). It sounds like that wasn't your take away from this discussion, so I'm happy to leave this as open discussion and put this on the next agenda to discuss further?

Reply:
Actually, we did have a vote in the last few seconds didn't we - so we decided on 1 - ALL of us. Let's write that instead.

Regarding 2 and the additional repo - I talked to the TAC about that and it's not a problem since openexr has already do this.

Reply:
I updated the wording here.